packages/ui/src/hooks/use-required-profile-error-handler.ts

```diff
@@ -42,6 +51,9 @@ const useRequiredProfileErrorHandler = (replace?: boolean) => {
             navigate(
               {
                 pathname: `/${UserFlow.continue}/email-or-phone/email`,
+                search: linkSocial
+                  ? `?${queryStringify({ [SearchParameters.linkSocial]: linkSocial })}`
```

Comment:
nit: Consider declaring a variable and set the queryStringify result to it, so that it could be re-used in these two places

Reply:
updated